NGUInjector/Managers/ZoneHelpers.cs

```diff
@@ -97,18 +97,25 @@ internal static void OptimizeITOPOD()
             var controller = Main.Character.adventureController;
             var level = controller.itopodLevel;
             var optimal = CalculateBestItopodLevel();
-            if (level == optimal) return;
-            controller.itopodStartInput.text = optimal.ToString();
-            if (optimal == Main.Character.adventure.highestItopodLevel)
+            if (optimal == Main.Character.adventure.highestItopodLevel && optimal != Main.Character.adventureController.maxItopodLevel())
             {
-                optimal++;
+                controller.itopodStartInput.text = optimal.ToString();
+                controller.itopodEndInput.text = Main.Character.adventureController.maxItopodLevel().ToString();
                 Main.Log("Pushed optimal floor up by 1 because highest floor is hit");
+                controller.verifyItopodInputs();
+                if (level !== optimal)
```

Comment:
So now you will have exactly same error as it have currently so it will reset it every 10 seconds..

Cause `level` will always be 1 lower than `optimal`

Reply:
Its still valid..